README.md

```diff
@@ -62,10 +65,19 @@ camke --build .
 OS/linux:
 make .
 ```
-- The executables will now be found within bin/debug
+- The executables will now be found within your desired build folder. bin/debug by default.
 - Note: You will also be able to open the VS project file via the *.sln* file within the build folder.
-- run the
+- Note: You may need to copy the models folder to the location of the executable as this is needed, especially if you changed the build directory.
+- Following you can run the executable such as Inter_SightPP.exe with the flags following.
+
+###flags
+Flags for running the system include:
+- "realsense" = run the system using a realsense camera
```

Comment:
We should probably make this a flag with a dash in front of it 😄 to match all the others, and I feel like -live or maybe like -live realsense if we we'll support other types of cameras in the future would be more descriptive.